python/units.py

```diff
@@ -0,0 +1,57 @@
+#!/opt/local/bin/python
```

Comment:
yeah, I wrote this at night and quickly.  I really need to fix setup.py to do this correctly.